uxarray/helpers.py

```diff
@@ -3,70 +3,75 @@
 from pathlib import PurePath
 
 
-# helper function to find file type
 def determine_file_type(filepath):
     """Checks file path and contents to determine file type. Supports detection
     of UGrid, SCRIP, Exodus and shape file.
 
-    Parameters: string, required
+    Parameters
+    ----------
+
+    filepath : :class:`string`: required
        Filepath of the file for which the filetype is to be determined.
 
-    Returns: string
-       File type: ug, exo, scrip or shp
+    Returns
+    -------
+    mesh_filetype : :class:`string`:
+        String describing the file type (ug, exo, scrip or shp)
+        to be used by Grid.py
 
     Raises:
        RuntimeError: Invalid file type
     """
     msg = ""
-    # exodus with coord
+    # Extract file name and extension, read into xarray.Dataset to prep
+    path = PurePath(filepath)
+    file_extension = path.suffix
```

Comment:
If the filepath is given somehow broken, there will be an exception that we won't catch.